benchmarks/setup/default/zeebe-values.yaml

```diff
@@ -127,7 +150,3 @@ kibana:
 # PROMETHEUS
 prometheus:
   enabled: false
-
-  servicemonitor:
-    enabled: true
```

Comment:
:question: We don't need the service monitor anymore? Does that mean the clusters will not be scraped by our Prometheus install?